src/constants/titles.js

```diff
@@ -0,0 +1,12 @@
+export const ErrorPagesTitles = {
+  FORBIDDEN_ERROR: "Forbidden",
+  ERROR_404: "Not Found 404",
+  NOT_CONNECTION: "No Connection",
+};
+
+export const ErrorPagesSubtitles = {
+  FORBIDDEN_ERROR:
+    "You are not authorized to see this resource, please login first",
```

Comment:
If this message is going to be displayed only when the user hasn't log in, it's ok. 
If It's thought to be displayed also when the user try to access to a page with an unauthorized role remove the last part as in the suggestion

```suggestion
    "You are not authorized to see this resource.",
```